packages/charting/src/components/LineChart/examples/LineChart.Basic.Example.tsx

```diff
@@ -138,6 +138,34 @@ export class LineChartBasicExample extends React.Component<{}, ILineChartBasicSt
             height={this.state.height}
             width={this.state.width}
             margins={margins}
+            legendProps={{ canSelectMultipleLegends: true, allowFocusOnLegends: true }}
```

Comment:
i am not able to select multiple legends, even though the api `canSelectMultipleLegends` is true.
please check and confirm from below url where the changes of this pr are hosted

http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/14597/merge/charting/dist/index.html#/examples/linechart

and also i request you to cut these changes and put in another example file which is `multiple line chart`.. there we have overflow  of the legends so i need to confirm how it behaves when over flow of legends happen

Reply:
![image](https://user-images.githubusercontent.com/69655270/93272918-75e93300-f76b-11ea-8739-8e2392f20e22.png)
It looks like the legendProps where not defined. I've updated the examples and waiting for the build to confirm.

Reply:
@shivasai09 I've updated the examples however it is not reflected in the test deployed site.

Reply:
i think it is deployed
![image](https://user-images.githubusercontent.com/33802398/93568518-ec757300-f9ad-11ea-8b2d-10212b058f42.png)


http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/14597/merge/charting/dist/index.html#/examples/linechart